internal/querynode/retrieve_collection.go

```diff
@@ -237,9 +237,45 @@ func (rc *retrieveCollection) doUnsolvedMsgRetrieve() {
 
 func (rc *retrieveCollection) retrieve(retrieveMsg *msgstream.RetrieveMsg) error {
 	// TODO(yukun)
+	retrieveResultMsg := &msgstream.RetrieveResultMsg{
```

Comment:
Don't forget set SealedSegmentIDsSearched, ChannelIDsSearched, GlobalSealedSegmentIDs later

Reply:
ok, currently it doesn't completely implemented. I will add these components later.